department-of-veterans-affairs/caseflow
# Move logstash config into its own file and fix user

### Description
This PR fixes logstasher user field and moves logstasher config to its own file for cleanliness 

### Acceptance Criteria

### Testing Plan
1. Deploy branch to UAT
2. Make sure app is healthy and user info shows up in logstasher logs

User info shows up and is searchable
![image](https://user-images.githubusercontent.com/3811786/117656824-d0a22580-b166-11eb-8b81-7465d3ae4380.png)
